doc/experiment.rst

```diff
@@ -12,316 +38,231 @@ Jupyter notebook, interactive python session, or Python file and use the
+be found or no launcher parameter is provided, the default value of
+`launcher="local"` will be used.
 
-If the launcher has been specified, or auto-detected through setting
-``launcher=auto`` in the Experiment initialization, the ``create_run_settings``
-method will automatically create the appropriate ``RunSettings`` object and
-return it.
+A `launcher` enables SmartSim to interact with the compute system
+programmatically through the Experiment python user interface.
+SmartSim users do not have to leave the Jupyter Notebook,
+Python REPL, or Python script to launch, query, and interact with their jobs.
 
-For example with Slurm
+To specify a specific launcher, one argument needs to be provided
+to the ``Experiment`` initialization.
 
-.. code-block:: Python
-
-  from smartsim import Experiment
-
-  exp = Experiment("hello_world_exp", launcher="slurm")
-  srun = exp.create_run_settings(exe="echo", exe_args="Hello World!")
-
-  # helper methods for configuring run settings are available in
-  # each of the implementations of RunSettings
-  srun.set_nodes(1)
-  srun.set_tasks(32)
-
-  model = exp.create_model("hello_world", srun)
-  exp.start(model, block=True, summary=True)
-
-  print(exp.get_status(model))
-
-The above will run ``srun -n 32 -N 1 echo Hello World!``, monitor its
-execution, and inform the user when it is completed. This driver script can be
-executed in an interactive allocation, or placed into a batch script as follows:
+.. code-block:: python
 
-.. code-block:: bash
+    from smartsim import Experiment
+
+    exp = Experiment("name-of-experiment", launcher="local")  # local launcher
+    exp = Experiment("name-of-experiment", launcher="slurm")  # Slurm launcher
+    exp = Experiment("name-of-experiment", launcher="pbs")    # PBSpro launcher
+    exp = Experiment("name-of-experiment", launcher="cobalt") # Cobalt launcher
+    exp = Experiment("name-of-experiment", launcher="lsf")    # LSF launcher
+    exp = Experiment("name-of-experiment", launcher="auto")   # auto-detect launcher
+
+
+=========
+ Entities
+=========
+
+Defining workflow stages requires the utilization of functions
+associated with the ``Experiment`` object. The ``Experiment`` object
+is intended to be instantiated once and utilized throughout
+the workflow runtime. In the following content, we define five
+entities created by an experiment: ``Orchestrator``, ``Model``, ``Ensemble``,
+``RunSettings``, and ``BatchSettings``, along with a discussion of their relationships.
+
+Entity Relationship Diagram
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Below is an Entity Relationship Diagram using Crow's Foot Notation.
+Moving from the top to the bottom of the diagram, you will first notice that
+an Experiment can have none or many ``Orchestrators``, ``Models``, and ``Ensembles``.
+The Experiment class is responsible for all entity initialization functions:
+
+1. ``Experiment.create_orchestrator()`` :raw-html:`&rarr; ``Orchestrator``
```

Comment:
The `raw-html` directive is not your best option here. I would suggest to add `..    include:: <isonum.txt>` at the top of the file, and then use `|rarr|` instead of ``:raw-html:`&rarr`` (which also misses the `` ` `` at the end)